src/main/java/teammates/common/util/AbsoluteUrl.java

```diff
@@ -0,0 +1,22 @@
+package teammates.common.util;
+
+/**
+ * An extension of the {@link Url} class which automatically appends the
+ * application's URL (as defined in {@link Config}) to the front of the URL input.
+ */
+public class AbsoluteUrl extends Url {
```

Comment:
Just my opinion: We may not need an entire new class to handle this functionality. We could just move the static function into the Url class.